composer.json

```diff
@@ -15,8 +15,8 @@
 		"soberwp/intervention": "1.2.0-p"
 	},
 	"require-dev": {
-		"humanmade/coding-standards": "^0.4.2",
-		"phpunit/phpunit": "~5.7",
+    "phpunit/phpunit": "~5.7",
```

Comment:
To avoid conflicts when working from `cd /srv/www/pressbooks.test/current/`

Please change to 6.5.8 (or ~6.5 or ... ?)

https://github.com/pressbooks/pressbooks/blob/dev/composer.json#L48